go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/sigstore/fulcio
 
-go 1.21.9
+go 1.22.2
```

Comment:
Let's split this out into its own PR, cc @cpanato

Reply:
https://github.com/sigstore/fulcio/pull/1706